magicmetamodel/src/main/kotlin/org/jetbrains/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/BspModuleDetailsToModuleTransformer.kt

```diff
@@ -26,16 +28,23 @@ internal class BspModuleDetailsToModuleTransformer(private val moduleNameProvide
       name = moduleNameProvider(inputEntity.target.id),
       type = inputEntity.type,
       modulesDependencies = buildTargetToModuleDependencyTransformer.transform(inputEntity.target),
-      librariesDependencies = DependencySourcesItemToLibraryDependencyTransformer
-        .transform(inputEntity.dependencySources.map {
-          DependencySourcesAndJavacOptions(it, inputEntity.javacOptions)
-        }),
+      librariesDependencies = calculateLibrariesDependencies(inputEntity),
       capabilities = inputEntity.target.capabilities.let {
         ModuleCapabilities(it.canRun, it.canTest, it.canCompile, it.canDebug)
       },
       languageIds = inputEntity.target.languageIds
     )
   }
+
+  private fun calculateLibrariesDependencies(inputEntity: BspModuleDetails): List<LibraryDependency> {
```

Comment:
i'd move this part to the python module transformer - just update the `librariesDependencies` there, and keep it 'normal' here